static/app/utils/replays/getCurrentUrl.tsx

```diff
@@ -16,15 +16,14 @@ function getCurrentUrl(
     crumb => crumb.type === BreadcrumbType.NAVIGATION
   ) as BreadcrumbTypeNavigation[];
 
-  const initialUrl = replayRecord.tags.url;
-  const origin = initialUrl ? new URL(initialUrl).origin : '';
```

Comment:
So I cherry picked this commit during hack week and we still need the original `origin`. Otherwise on https://github.com/getsentry/sentry/pull/38092/files#diff-b3b1434537e36c425cc3761c3ed1c3f57bd4ee3830009f1806ec6a8f735ad304R35 it will preserve the file path.